app/src/main/java/com/hedvig/app/feature/keygear/ui/tab/KeyGearFragment.kt

```diff
@@ -40,6 +38,11 @@ class KeyGearFragment : BaseTabFragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
+        keyGearRoot.updatePadding(top = getToolbarBarHeight(this))
```

Comment:
What if the toolbar height changes at runtime? Will this padding be correct, then?

Don't we need to do something similar to the `bottomTabInset` from the `LoggedInViewModel`?